ang/civicase/ActivityFeed.js

```diff
@@ -60,8 +59,18 @@
     /**
      * Refresh Activities
      */
-    $scope.refreshAll = function () {
-      getActivities(false, $scope.refreshCase);
+    $scope.refreshAll = function (apiCalls) {
+      if (_.isFunction($scope.refreshCase)) {
```

Comment:
Can we add a bit documentation to the logic which make sense to a layman. The current logic checks if the function `refreshCase` resides inside the scope then it will be called else get activities will be called following the api calls. But this doesn't make much sense.

Reply:
Added doc in the code.